config/parameters.yml

```diff
@@ -36,6 +36,8 @@ aws_s3:
   region: ""
   bucket_name: ""
 
+header_extra_options: 'User-Agent: Mozilla/5.0 (Windows; U; Windows NT 6.1; rv:2.2) Gecko/20110201'
```

Comment:
could you add a comment or reference to the issue?

or add it in the docs

Reply:
Good catch! I'll add comments above it.